files/zh-cn/_redirects.txt

```diff
@@ -1098,7 +1098,7 @@
 /zh-CN/docs/Learn/JavaScript/Asynchronous/Async_await	/zh-CN/docs/conflicting/Learn/JavaScript/Asynchronous/Promises
 /zh-CN/docs/Learn/JavaScript/Asynchronous/Choosing_the_right_approach	/zh-CN/docs/conflicting/Learn/JavaScript/Asynchronous
 /zh-CN/docs/Learn/JavaScript/Asynchronous/Concepts	/zh-CN/docs/conflicting/Learn/JavaScript/Asynchronous/Introducing
-/zh-CN/docs/Learn/JavaScript/Asynchronous/Timeouts_and_intervals	/zh-CN/docs/conflicting/Learn/JavaScript/Asynchronous_ae5a561b0ec11fc53c167201aa8af5df
```

Comment:
For those four redirects:

| origin | redirect |
| ------ | --------- |
| /zh-CN/docs/Learn/JavaScript/Asynchronous/Async_await | /zh-CN/docs/Learn/JavaScript/Asynchronous/Promises |
| /zh-CN/docs/Learn/JavaScript/Asynchronous/Choosing_the_right_approach | /zh-CN/docs/Learn/JavaScript/Asynchronous |
| /zh-CN/docs/Learn/JavaScript/Asynchronous/Concepts | /zh-CN/docs/Learn/JavaScript/Asynchronous/Introducing |
| /zh-CN/docs/Learn/JavaScript/Asynchronous/Timeouts_and_intervals | /zh-CN/docs/Learn/JavaScript/Asynchronous |